store/kv.go

```diff
@@ -8,27 +8,27 @@ import (
 	"path/filepath"
 	"time"
 
-	"github.com/dgraph-io/badger/v3"
 	ds "github.com/ipfs/go-datastore"
 	dsq "github.com/ipfs/go-datastore/query"
-	badger3 "github.com/ipfs/go-ds-badger3"
+
+	badger4 "github.com/celestiaorg/go-ds-badger4"
```

Comment:
re: https://github.com/rollkit/rollkit/pull/1419#issuecomment-1865168289

still uses celestiaorg fork, can we switch this to use badger's fork?

Reply:
oh, i thought it was about the underlying library, not the ds wrapper. Ok

Reply:
our ds is forked from [this](https://github.com/ipfs/go-ds-badger3). Lemme see if it works